gr-channels/lib/flat_fader_impl.cc

```diff
@@ -38,6 +35,12 @@ flat_fader_impl::flat_fader_impl(uint32_t N, float fDTs, bool LOS, float K, uint
       scale_los(sqrtf(d_K) / sqrtf(d_K + 1)),
       scale_nlos(1 / sqrtf(d_K + 1))
 {
+    d_theta = dist_1(rng_1);
```

Comment:
rng_1 is used here but is set below. Is that intentional?

Reply:
yes, my mistake, I have pushed a fix to reverse it.